source/_integrations/modbus.markdown

```diff
@@ -613,7 +613,8 @@ climates:
           required: true
           type: integer
         write_registers:
-          description: "Request type, use `write_registers` if true  else `write_register`."
+          description: "Request type, use `write_registers` if true  else `write_register`.
+            If more than one value is specified for a specific mode, only the first one is used for writing to the register."
```

Comment:
Which mode, there are no explanation of how to set multiple modes or multiple values.

Reply:
Ahh sorry, copy and paste...